Makefile

```diff
@@ -17,11 +17,12 @@ endif
 ifeq ("$(IN_DOCKER)","TRUE")
 	DOCKER_RUN=
 else
+	PHP_VERSION=$(shell docker run --rm -v "`pwd`:`pwd`" jess/jq jq -r -c '.config.platform.php' "`pwd`/composer.json" | php -r "echo str_replace('|', '.', explode('.', implode('|', explode('.', stream_get_contents(STDIN), 2)), 2)[0]);")
```

Comment:
You may want to use the `:=` operator instead to only execute that expression once.

More info: http://www.gnu.org/software/make/manual/html_node/Flavors.html#Flavors

Reply:
Good one @lcobucci, updated it, thanks 👍!